packages/client/utils/AtlassianManager.ts

```diff
@@ -743,6 +744,16 @@ export default abstract class AtlassianManager {
       res.message.startsWith(timeTrackingFieldName ? timeTrackingFieldId : fieldId) &&
       res.message.includes('is not on the appropriate screen')
     ) {
+      const projectKey = JiraProjectKeyId.join(issueKey)
+      const project = await this.getProject(cloudId, projectKey)
```

Comment:
-1 if there's an Error (e.g. the project can't be found or the rate limit was hit) we want to use that error instead.

to fix, add this below:

```ts
if (project instanceof RateLimitError || project instanceof Error) {
        throw project
      }
```

after that, you can remove `'simplified' in project &&` below because we're guaranteed it's a project.